src/Telegram/Attributes/BaseAttribute.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace SergiX44\Nutgram\Telegram\Attributes;
+
+use ReflectionClass;
+
+abstract class BaseAttribute
```

Comment:
```suggestion
abstract class BaseEnum
```
In the future may be converted to enums